python/hsfs/util.py

```diff
@@ -127,7 +128,7 @@ def check_timestamp_format_from_date_string(input_date):
     return normalized_date, date_format
 
 
-def get_timestamp_from_date_string(input_date, time_zone=None):
+def get_timestamp_from_date_string(input_date, time_zone=timezone.utc):
```

Comment:
Do we need to document this?

Reply:
I don't think we need to change/update documentation at this point. Since this is just a measure to fix an existing inconsistency (at some points we interpret user input as utc, in others not). David is working on a more comprehensive PR, which fixes this and related issues. Right now only users using external client in different timezones and calling a specific subsets of functions would be impacted by this change. We should consider adding a guide around timestamp/timezones at some point.

Reply:
If you prefer I can set the parameter only explicitly for the calls of `util.get_timestamp_from_date_string()` within `as_of`.